CMakeLists.txt

```diff
@@ -4,15 +4,15 @@ project("gpu-filters")
 # -- Config -- #
 set(DEBUG_FLAGS -"D_GLIBCXX_DEBUG")
 set(CMAKE_C_FLAGS "-std=c99")
-set(CMAKE_CXX_FLAGS "-std=c++11 -Wconversion -Wshadow -Wall -Wextra -Wno-missing-braces -DCL_HPP_TARGET_OPENCL_VERSION=200 -DCL_HPP_MINIMUM_OPENCL_VERSION=200")
+set(CMAKE_CXX_FLAGS "-std=c++11 -Wconversion -Wshadow -Wall -Wextra -Wno-missing-braces -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_HPP_MINIMUM_OPENCL_VERSION=120")
 # https://stackoverflow.com/questions/44942225/opencl-vector-types-cant-access-unioned-components-x-y-z-with-c11-enabled
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -U__STRICT_ANSI__")
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 # set(CMAKE_VERBOSE_MAKEFILE on)
 
 # -- Libraries -- #
-find_package( OpenCV REQUIRED )
+find_package( OpenCV REQUIRED highgui imgproc core )
```

Comment:
Just built locally and it seems I also need `imgcodecs` and `videoio`. What OpenCV version are you running? Mine is 3.2. 

If it's a versioning thing, I think we should ask for a specific one to avoid these problems, like so: `find_package(OpenCV 1.1 REQUIRED ..)`. Same for OpenCL

Reply:
No no, I think they are also needed for the video_input part. It didn't complain in my case cause only ran the `comparison_tests` target. Will update that

Reply:
Oh, I installed OpenCV2 because the README said so. Will check this, because OpenCV 2 doesn't have the `imgcodecs` and `videoio` libs.

Reply:
You are right, my bad. We are not using any 2 nor 3 specific features though, so no big deal, but we should make everything consistent and update the readme if needed. I would stick to the newer version 3.2, since that's what `sudo apt-get install libopencv-dev` is installing now.